nack/receiver_interceptor.go

```diff
@@ -153,6 +153,7 @@ func (n *ReceiverInterceptor) loop(rtcpWriter interceptor.RTCPWriter) {
 }
 
 func nackPairs(seqNums []uint16) []rtcp.NackPair {
+	// TODO: I think this shoud be moved to rtcp package
```

Comment:
Agreed.  The function I'm using for that in the Unnamed SFU has the following shape:
```
func ToBitmap(seqnos []uint16) (first uint16, bitmap uint16, remain []uint16)
```
it returns the first NackPair (as first/bitmap) and anything remaining afterwords.  You're supposed to loop until `remain` is empty.

Reply:
Moved! `NackPairsFromSequenceNumbers` is now in `pion/rtcp`